schemas/core/common_test.go

```diff
@@ -61,35 +61,27 @@ func TestCommonValidation(t *testing.T) {
 		require.Equal(t, failtags[e], err.ActualTag())
 	}
 
-	// Wrong URN fail on urn validation tag, right ID
-	c = CommonAttributes{
-		Schemas: []string{"not-a-urn"},
-		ID:      "test",
-	}
+	// Empty Schema fail on gt validation tag, ID fails on excludes validation tag
+	c.Schemas = []string{}
+	c.ID = "bulkId"
 	errors = validation.Validator.StructExcept(c, "Meta")
+	require.Len(t, errors, 2)
+
 	for _, err := range errors.(validator.ValidationErrors) {
 		require.NotNil(t, err)
 		require.IsType(t, (validator.ValidationErrors)(nil), errors)
 	}
-	require.Len(t, errors, 1)
 
-	// Empty Schema fail on gt validation tag, ID fails on excludes validation tag
-	c = CommonAttributes{
-		Schemas: []string{},
-		ID:      "bulkId",
-	}
+	// Wrong URN fail on urn validation tag, right ID
```

Comment:
Ok, please improve comment writing `valid ID`. It is more readable and understandable.